### PR TITLE
added basic profanity filter 

### LINKED ISSUE #1155: Profanity Filter in ChatApp
**Actual Behaviour**

I wanted to explore a way to handle Cuss Words

**Expected Behaviour**

To implement a way to handle clean chats

**Screenshots of the issue**
screenshot from 2018-03-15 23-08-30

**Would you like to work on the issue?**

Yes I would definitely love to work on this

### PR DESCRIPTION
**Pull request by KrUciFieR-Jr**
Fixes #1155 

Changes: Added a basic profanity filter and also created a whitelist which would declassify the wrong mentions 

Demo Link: https://same-bee.surge.sh

Screenshots for the change: 
screenshot from 2018-03-16 01-16-25

### CODE REVIEW

**Comment by KrUciFieR-Jr** [comment]
Please check for review looking forward to expand this feature

File: package.json [hunk 1]
```diff
@@ -17,18 +17,19 @@
...
+    "material-ui-responsive-menu": "^1.19.3",
     "material-ui-scrollable-tabs": "^1.0.0",
     "moment": "^2.18.1",
-    "prop-types": "^15.5.10",
-    "react": "^15.5.4",
+    "prop-types": "^15.6.1",
+    "react": "^15.6.2",
     "react-bootstrap": "^0.31.0",
     "react-color": "^2.12.1",
+    "react-container-dimensions": "^1.3.3",
```
Comment (duttaditya18) [inline package.json]: Why this?

**Comment by Orbiter** [comment]
Profanity must be handled on server side

**Comment by duttaditya18** [comment]
@Orbiter @KrUciFieR-Jr What is the point of a profanity filter anyway?

**Pull request closed without merging**